EzMap.Domain/Models/Collection.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EzMap.Domain.Models;
+
+public class Collection : EntityBase<Guid>
```

Comment:
Super generic name, at least make it `PoiCollection`. This also collides with ICollection in BCL, so not really recommended.

Reply:
Sharing same thought, were not able to come up a better name

Reply:
Now I'm lost, why are there both Collection and Itinerary.